contracts/src/RedeemManager.1.sol

```diff
@@ -72,17 +71,14 @@ contract RedeemManagerV1 is Initializable, IRedeemManagerV1, IProtocolVersion {
 
     function _redeemQueueMigrationV1_2(address[] memory _prevInitiators) internal {
         RedeemQueueV1.RedeemRequest[] memory initialQueue = RedeemQueueV1.get();
```

Comment:
I think we should rename it to `oldQueue` to make it more readable. It is easier to follow the code the when we have old & new vs initial & new.

Reply:
Makes sense, would update.